musit-models/src/main/scala/no/uio/musit/models/GroupModule.scala

```diff
@@ -0,0 +1,37 @@
+package no.uio.musit.models
+
+import play.api.libs.json._
+
+sealed trait GroupModule {
+  val id: Int
+}
+
+case object StorageFacility extends GroupModule { val id = 1 }
+case object Loan extends GroupModule { val id = 2 }
+case object Analysis extends GroupModule { val id = 3 }
+
```

Comment:
for `GodMode` (and other super users, e.g. `MusitAdmin`) it's good to have an `All` here as well.